torch/csrc/jit/codegen/cuda/lower_unroll.h

```diff
@@ -98,7 +100,6 @@ class TORCH_CUDA_API UnrollPass : public OptOutDispatch {
   static std::vector<Expr*> runPass(
       Fusion* fusion,
       const std::vector<Expr*>& exprs,
-      const std::unordered_set<Expr*>& init_exprs,
```

Comment:
This seems to remove a recent fix on predicates for initializing local buffers. See #246 and #255.

Reply:
We can re-enable that fix without carrying init_exprs. We can detect initialization expressions by their pattern without carrying them around explicitly.

Reply:
Could you open a new issue and I can do a follow up PR? I believe we still have correctness now.

Reply:
You're right. The previous failing test still works fine, but I'm not sure whether it's intended behavior. I re-opened the issue (#64).